swarm/README.md

```diff
@@ -0,0 +1,135 @@
+# Deploy using docker swarm
+
+This document guides you to deploy a cluster using docker swarm with physical nodes.
+
+## Pre-requisite 
+
+- Ubuntu 22.04
+- [Docker](https://docs.docker.com/engine/install/ubuntu/#install-using-the-repository)
+
+## Build docker images on all nodes
+
+Ensure that **grpc_python:dev** and **cam-sim:dev** docker images are built on every node.
+You can transfer the docker images to the other nodes or you can clone and run this command on every node to ensure the images are available.
+
+Run these commands at the project's root level.
+
+```
+make build-grpc_python
+```
+
+Download models and media files:
+
+```
+make prepare-inputs
+```
+
+## Setting up Docker swarm
+
+Start swarm mode
+
+```
+docker swarm init
+```
+
+### Join other worker nodes
+
+For now, only the manager node will be available, if you want to join other nodes, execute the following command:
+
+```
+docker swarm join-token worker
+```
+
+This command will output something like the following, including a docker swarm join command with the token:
+
+```
+To add a worker to this swarm, run the following command:
+
+    docker swarm join --token SWMTKN-1-49nj1cmql0... 192.168.99.100:2377
```

Comment:
even though it's just a partial token some tools might flag it as having a token in the repo. I would suggest something like <join-token> to replace the token.